### DISCUSSION TITLE
How can I make Fooocus utilize 100% of the GPU resources?

### DISCUSSION THREAD

**Discussion by Jmdatabase** [lllyasviel/Fooocus#3784] [Q&A] [answered]
My graphics card is an RTX 3080Ti, but when running FOOOCUS, the GPU VRAM usage is only around 6GB, never exceeding 10GB. Where can I configure FOOOCUS to use more VRAM?

Thanks a lot! 
@_@

**Comment by XDM-Inc** [comment, marked as answer]
im pretty sure it will use what it needs to. i have a 7900 XTX and average usage is about 11-13gb for regular use but if i use certain LX or High models it will max out my 24gb and lockup my pc. You can try start parameter  "--always-high-vram"  at your own risk as you might max out vram and crash and/or lockup your pc
